### PR TITLE
#32 Remove deprecated classes and functions.

### PR DESCRIPTION
**Pull request by sallner**
Let the deprecations vanish. 

We still have failing tests with Python 3.7 on travis which might be because of 3.7.1. Locally with 3.7.8 I cannot reproduce the error.

### CODE REVIEW

File: src/AccessControl/Owned.py
Comment (dataflake) [inline src/AccessControl/Owned.py:22]: That's also obsolete. I'd add a deprecation for version 6 here. I don't know if it's safe to remove.

File: src/AccessControl/Role.py
Comment (dataflake) [inline src/AccessControl/Role.py:26]: These are als obsolete, see above

File: src/AccessControl/User.py
Comment (dataflake) [inline src/AccessControl/User.py:36]: Also obsolete, see above

**Review by dataflake — CHANGES_REQUESTED** [review]
There are several modules that seem to consist of BBB imports only at this point. Those should get deprecation warnings for version 6 so we can delete them later.

**Comment by sallner** [comment]
> There are several modules that seem to consist of BBB imports only at this point. Those should get deprecation warnings for version 6 so we can delete them later.

That is actually done in https://github.com/zopefoundation/AccessControl/pull/102, which is the reason I submitted this PR as second one. Or did I miss something?

**Comment by dataflake** [comment]
The deprecation warnings point to a removal in version 6. Now you remove them in version 5?

**Comment by sallner** [comment]
With this PR the already deprecated features are removed in version 5, as they state that in there message. With #102 new deprecation warnings are introduced, with the removal planned in version 6.

**Comment by dataflake** [comment]
I think it would be helpful to merge the first PR before asking for opinions on the second. This is confusing.

**Comment by sallner** [comment]
I am happy to merge this PR before #102, if you approve the PR. The reason I submitted #102 first, was that if those deprecations should already be happen with version 5, this had to be merged and released as a version 4.3 before the merge of this PR #103.

**Comment by dataflake** [comment]
I already approved #102

**Review by icemac — APPROVED** [review]

**Comment by icemac** [comment]
@dataflake I merged #102 and updated this PR to `master`. Is is okay for you now?

**Review by dataflake — APPROVED** [review]

**Pull request merged**
